.travis.yml

```diff
@@ -18,10 +18,12 @@ matrix:
     - php: 5.6
       env: DB=MYSQL PDO=1
     - php: 5.6
-      env: DB=MYSQL BEHAT_TEST=1
-    - php: 5.6
+      env: DB=MYSQL BEHAT_TEST=1 CMS_TEST=1
```

Comment:
Did you mean to remove the framework behat tests here? You're running cms behat tests twice now.